cmd/cri-containerd/options/options.go

```diff
@@ -179,17 +167,41 @@ func (c *CRIContainerdOptions) InitFlags(fs *pflag.FlagSet) error {
 	// But the priority of the toml config value is higher than the default value,
 	// Without a way to insert the toml config value between the default value and the command line value.
 	// We parse twice one for default value, one for commandline value.
```

Comment:
If we just need one parse, The comment is outdated.

Reply:
The first parse is done by cobra, so actually there are still 2 parse.

Reply:
I see.   cobra.Command.Execute() do it. 
Thanks